### PR TITLE
Fix/start value for historical_forecasts in examples

### PR DESCRIPTION
**Pull request by aarondt1**
- Update start value from 0.6 to 0.75 to match the text above the cell.
- Fixing typos

The text above the cell reads: 

> We can also backtest the models using covariates. Say for instance we are interested in evaluating the running accuracy with a horizon of 12 months, starting at 75% of the air series

This commit resolves the issue where the start value was incorrectly set to 0.6 instead of the intended 0.75.

### CODE REVIEW

**Review by dennisbader — CHANGES_REQUESTED** [review]
Thanks for this 🚀  

Could you rather change the text to 60% so that we don't have to rerun the example notebook?

**Comment by aarondt1** [comment]
> Thanks for this 🚀
> 
> Could you rather change the text to 60% so that we don't have to rerun the example notebook?

Sure thing!

File: examples/01-multi-time-series-and-covariates.ipynb [hunk 1]
```diff
@@ -758,13 +758,14 @@
    ]
   },
   {
+   "attachments": {},
```
Comment (dennisbader) [inline examples/01-multi-time-series-and-covariates.ipynb:761]:
```suggestion
```

File: examples/01-multi-time-series-and-covariates.ipynb [hunk 2]
```diff
@@ -848,6 +849,7 @@
...
```
Comment (dennisbader) [inline examples/01-multi-time-series-and-covariates.ipynb:852]:
```suggestion
```

**Review by dennisbader — APPROVED** [review]
LGTM, thanks!

**Pull request merged**